File: src/harkerrobolib/auto/SequentialCommandGroup.java
package harkerrobolib.auto;

import edu.wpi.first.wpilibj.command.Command;
import edu.wpi.first.wpilibj.command.CommandGroup;

/**
 * 
 * @author Finn Frankis
 * @version 10/18/18
 */
public class SequentialCommandGroup extends CommandGroup {
	public SequentialCommandGroup (Command... commands) {
		for (Command c : commands) {addSequential(c);}
	}
}

File: src/harkerrobolib/auto/SequentialCommandGroup.kt
package harkerrobolib.auto

import edu.wpi.first.wpilibj.command.Command
import edu.wpi.first.wpilibj.command.CommandGroup

/**
 *
 * @author Finn Frankis
 * @version 10/18/18
 */
class SequentialCommandGroup(vararg commands: Command) : CommandGroup() {
    init {
        for (c in commands) {
            addSequential(c)
        }
    }
}
